exporter/redis.go

```diff
@@ -182,6 +182,11 @@ func (e *Exporter) initGauges() {
 		Name:      "command_call_duration_seconds_sum",
 		Help:      "Total amount of time in seconds spent per command",
 	}, []string{"addr", "alias", "cmd"})
+	e.metrics["slowlog_len"] = prometheus.NewGaugeVec(prometheus.GaugeOpts{
+		Namespace: e.namespace,
+		Name:      "slowlog_len_sum",
+		Help:      "Total slowlog",
+	}, []string{"addr", "alias", "cmd"})
```

Comment:
The `cmd` label isn't needed. See also below.

Reply:
just put it for ... funny😊

Reply:
:-)